src/workers/userCompanyApprovedCio.ts

```diff
@@ -0,0 +1,35 @@
+import { TypedWorker } from './worker';
+import { Company } from '../entity/Company';
+import { cioV2, identifyUserCompany } from '../cio';
+
+const worker: TypedWorker<'api.v1.user-company-approved'> = {
+  subscription: 'api.user-company-approved-cio',
+  handler: async (message, con, log) => {
+    if (!process.env.CIO_SITE_ID) {
+      return;
+    }
+
+    const {
+      data: { userCompany },
+    } = message;
+
+    if (!userCompany.companyId) {
+      return;
+    }
+
+    const company = await con
+      .getRepository(Company)
+      .findOneBy({ id: userCompany.companyId });
+    if (!company) {
+      log.info(
```

Comment:
`warn`, not sure if we are expecting it?

Reply:
hopefully not :)

Reply:
I mean it should be warn if its not supposed to happen often so we se it in logs more easily